### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ashoktelukuntla**
### Description
Changed MAINTAIERS.md, added CONTRIBUTING.md,DEVELOPER_GUIDE.md,LICENCE

### Issues Resolved

Resolving Issue#53

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (achitojha) [inline LICENSE:1]: I am not sure if we can add this here. The data used in each workload comes with its own license and I think we may need to make sure we're not overriding the licenses here.

Each workload had its own license which resides at the end of each workload's README file. See example : https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/eventdata/README.md

File: DEVELOPER_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Developer Guide
```
[the remark this answers is not in the document]
Comment (ashoktelukuntla) [reply DEVELOPER_GUIDE.md:1]: Issue#53 - Is asking for standardizing MAINTAIERS.md and other file -which has links to Developer guide. So need to add the same

File: LICENSE (hunk 1 above)
Comment (ashoktelukuntla) [reply LICENSE:1]: I remove this file and raise another PR

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    - [First Things First](#first-things-first)
+    - [Ways to Contribute](#ways-to-contribute)
+        - [Bug Reports](#bug-reports)
+        - [Feature Requests](#feature-requests)
+        - [Documentation Changes](#documentation-changes)
+        - [Contributing Code](#contributing-code)
+    - [Developer Certificate of Origin](#developer-certificate-of-origin)
+    - [Review Process](#review-process)
+
+# Contributing to OpenSearch Dashboards
```
[the remark this answers is not in the document]
Comment (ashoktelukuntla) [reply CONTRIBUTING.md:11]: Good catch - will make the changes

**Pull request closed without merging**